jekyll/_cci2/github-apps-integration.adoc

```diff
@@ -0,0 +1,255 @@
+
+Follow the steps on the xref:first-steps#[Sign up and try CircleCI] page to connect your GitHub account, which creates your CircleCI organization.
+
+When you create a new organization and connect your GitHub account, you will be prompted to create a new project from a repository. You will be presented with the following options to set up a CircleCI configuration for your project:
+
+* **Faster**: Commit a starter CI pipeline to a new branch.
++
+CircleCI automatically creates a new branch and commits a basic configuration file. You may make further changes to the file afterwards in your GitHub repo.
+
+* **Fast**: Use a `config.yml` template that you can edit and save.
++
+You can choose a sample `.circleci/config.yml` to best fit your project from a variety of templates (for example, Node.js, Python, iOS apps). You can make changes to the file in CircleCI before saving. The config is committed on a new branch.
+
+Choosing **Faster** automatically triggers a pipeline once you create the project.
+
+NOTE: If you are new to CircleCI, you may wish to get started with our xref:getting-started#[Quickstart guide], our xref:hello-world#[Hello world] examples, or take a look at some of our xref:sample-config#[sample configurations] that demonstrate different types of workflows. The xref:configuration-reference#[Configuration reference] is a full reference to the keys used in a `.circleci/config.yml` file.
+
+[#trigger-pipeline]
+== Trigger a pipeline in CircleCI
+
+When you create a new project using the **Faster** (commit a starter CI pipeline) option described in the section above, a pipeline is automatically triggered. You should see the pipeline running shortly after you are taken to the CircleCI dashboard.
+
+If you use the **Fast** config setup, the pipeline is not triggered until you save the `.circleci/config.yml` by clicking the **Commit and Run** button in the web app.
+
+Each time you push changes to your GitHub repository, a new pipeline is triggered and you should see it running for the project within the CircleCI web app.
+
+image::{{site.baseurl}}/assets/img/docs/gl-ga/gitlab-ga-successful-pipeline.png[Successful pipeline run]
+
+Editing an existing CircleCI configuration within the web app is not currently available. You may make further changes to the config in your GitHub repo. Committing further changes in your repo will automatically trigger a pipeline.
+
+Manually triggering a pipeline from the CircleCI web app is not available at this time.
+
+[#project-settings]
+== Project settings
+
+Within CircleCI, a project can have one or more **configurations**, which are pipeline definitions. _Configurations_ include, but are not limited to, a `.circleci/config.yml` file in your repo.
+
+A project can have one or more **triggers**, which are events from a source of change. _Triggers_ include, but are not limited to, a VCS. A trigger determines which configuration should be used to start a pipeline.
```

Comment:
I don't really know how to address this: "A trigger determines which configuration should be used to start a pipeline".

This is _technically_ true of GitHub Apps, but is not true of GitLab. A trigger will reference a configuration source, and the configuration source determines which configuration is used to start a pipeline.

We will be building this concept out for GitHub Apps, and introducing it post-launch (I don't have a timeline on it).

Maybe best to leave as is for now because it's accurate to the current state of GitHub Apps?

Is there a process for updating this when we are able to add support for config sources?

Reply:
Any customer-facing changes would ideally be passed to the docs team to update relevant pages so as long as we get notified, preferably in advance, we can handle the updates.

Reply:
I'll leave this as it is for now.